pages/index.js

```diff
@@ -0,0 +1,357 @@
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import styles from "../styles/Home.module.css";
+import Header from "../components/Header";
+import ClientLibraries from "../components/ClientLibraries";
+
+export default function Home() {
+  const [latestPosts, setLatestPosts] = useState();
+
+  const fetchDevToPosts = async () => {
+    const data = await fetch(
+      "https://dev.to/api/articles?username=stripe&state=all"
+    );
+    const posts = await data.json();
+
+    const latest = posts.slice(0, 3);
+
+    setLatestPosts(latest);
+  };
+
+  useEffect(() => {
+    fetchDevToPosts();
+  }, []);
+
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Stripe Developer Resources</title>
+        <link
+          href="https://fonts.googleapis.com/css?family=Source+Code+Pro"
+          rel="stylesheet"
+        />
+        <link rel="shortcut icon" href="favicon.ico" />
```

Comment:
Favicon isn't rendering for me, I _think_ that will also be solved when images are moved to `/public/`

Reply:
The issue with the images is that running the app with `npm run dev` needs to find the images in the `public` folder but once built, they need to be at the root of the project, directly in the `images` folder :/ so for now i duplicated the folder until we can move away from GitHub pages. If we host this site on Netlify or Vercel, it will be a lot easier.

Reply:
This is something we'd have to run locally right? Or do you mean adding it somewhere in the project?